src/universe/qual/Bottom.java

```diff
@@ -13,8 +13,8 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
-
-import javax.lang.model.type.TypeKind;
+import java.math.BigDecimal;
```

Comment:
Should these two classes be listed in the types for which to use bottom?

Reply:
Was added in DefaultFor previously (https://github.com/opprop/universe/commit/c1e9c711f9172649aa0d1a91fa7637b3ce94a210), but was removed again later.